WtWSMS/common/culture/creation_names/BP_ck2_custom_names.txt

```diff
@@ -0,0 +1,238 @@
+﻿#The most glorious of all cultures, East Celtic
+east_celtic = {
+	trigger = {
+		has_valid_east_celtic_ancestors_trigger = {
+			CULTURE_1 = scope:culture
+			CULTURE_2 = scope:other_culture
+		}
+	}
+	hybrid = yes
+}
+##### THIS SECTION BASED ON CK2/WtWSMS/events/WtWSMS_culture_conversion_events.txt
```

Comment:
The old system which was disabled? Why is this used as the basis as opposed to the more elaborate WtWSMS_melting_pots_events.txt?